optimum/exporters/onnx/convert.py

```diff
@@ -1196,6 +1201,20 @@ def onnx_export_from_model(
         optimization_config.disable_shape_inference = True
         optimizer.optimize(save_dir=output, optimization_config=optimization_config, file_suffix="")
 
+    if slim:
+        if not is_onnxslim_available():
+            raise ImportError("The pip package `onnxslim` is required to optimize onnx models.")
+
+        import glob
+
+        from onnxslim import slim
+
+        onnx_models = glob.glob(os.path.join(output, "**", "*.onnx"), recursive=True)
+
+        for model in onnx_models:
+            slimmed_model = slim(model)
+            check_and_save_model(slimmed_model, model)
```

Comment:
I just noticed we already have paths in onnx_file_subpaths variable and it's more robust

```suggestion
        import glob

        from onnxslim import slim

        for onnx_file_path in onnx_files_subpaths:
            slimmed_model = slim(onnx_file_path)
            check_and_save_model(slimmed_model, onnx_file_path)
```